bindings/wasm/bindings.ts

```diff
@@ -0,0 +1,43 @@
+type Matrix1x3 = [Number, Number, Number];
+type Matrix4x3 = [Matrix1x3, Matrix1x3, Matrix1x3, Matrix1x3];
+type Vec3 = [Number, Number, Number];
+type Vec3Fun =
+    ((v: Vec3) => Manifold)&((x: Number, y?: Number, z?: Number) => Manifold);
+type Point = [Number, Number];
+type SimplePolygon = Point[];
+type Polygon = SimplePolygon[];
+
+declare class Manifold {
+  transform(matrix: Matrix4x3): Manifold;
+
+  translate: Vec3Fun;
+  rotate: Vec3Fun;
+  scale: Vec3Fun;
+  add(other: Manifold): Manifold;
+  subtract(other: Manifold): Manifold;
+  intersect(other: Manifold): Manifold;
+  refine(n: Number): Manifold;
+}
+
+declare function cube(size: Vec3, center?: boolean): Manifold;
+declare function cube(
+    x: Number, y?: Number, z?: Number, center?: boolean): Manifold;
```

Comment:
`size: Vec3`?

Reply:
Nice catch, the binding doesn't allow this to be optional right now. Will fix.